packages/preview-middleware/templates/test/qunit.html

```diff
@@ -12,7 +12,7 @@
 		}'
 		data-sap-ui-animation="false"
 		data-sap-ui-compatVersion="edge"
-		data-sap-ui-async="true"
+		data-sap-ui-async="false"
```

Comment:
Is this change related to proxying the ui5 sources? Never had issues with that one when using the sources served by tooling via npmjs

Reply:
I cannot explain it to you but I currently assume that the proxy is faster and thats why something is loaded to early. I am not accepting `false` here, I am looking for a better option. Also started to pull in  @petermuessig but it isn't clear yet why things are happening.

Reply:
the initial qunit lib needs to be loaded synchronously, so I moved it into an html script tag. With that I can keep true here.